SU2_CFD/include/solvers/CIncEulerSolver.hpp

```diff
@@ -395,8 +395,30 @@ class CIncEulerSolver : public CFVMFlowSolverBase<CIncEulerVariable, ENUM_REGIME
   inline bool GetHasHybridParallel() const final { return true; }
 
   /*!
-   * \brief Get values for streamwise periodic flow: delta P, m_dot, inlet T, integrated heat.
-   * \return Struct holding 4 su2doubles.
+   * \brief Get values for streamwise periodic flow: delta P, m_dot, inlet T, integrated heat, etc.
+   * \return Struct holding streamwise periodic values.
    */
-  StreamwisePeriodicValues GetStreamwisePeriodicValues() const final { return SPvals; }
+  StreamwisePeriodicValues GetStreamwisePeriodicValues() final { return SPvals; }
+  StreamwisePeriodicValues GetStreamwisePeriodicValuesUpdated() final { return SPvalsUpdated; }
```

Comment:
Do you also need the updated ones? Or can that be an internal detail since SPvals is set to Updated in normal conditions?

Reply:
That method is not necessary anymore indeed :+1: